BurnerB/FreeMentors-V2
# [#168093938] Fix maintainability

#### What does this PR do?
Fix poor maintainability rating
#### Description of Task to be completed?
* Add the build folder to gitignore
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch bg-fix-maintainability-168093938
* Run npm --init to install all dependancies.
* Run command npm test on your terminal
* Tests should pass
* Maintainability rating should be higher on the readme badge

#### What are the relevant pivotal tracker stories?
[#168093938](https://www.pivotaltracker.com/story/show/168093938)
#### Screenshots (if appropriate)
